docs/user/downloadable-documentation.rst

```diff
@@ -1,50 +1,42 @@
-Downloadable Documentation
-==========================
+Offline Formats
+===============
 
-Read the Docs supports building multiple formats for Sphinx-based projects:
+Read the Docs supports building multiple offline formats:
```

Comment:
I guess we should remove "Sphinx-based" because the build process can be configured for any kind of documentation stack?

Should we clarify in a separate section that the easiest integration happens with Sphinx, but we support any kind of build process and making different formats available is a matter of moving build artifacts into the correct location at the end of a build?

Reply:
> I guess we should remove "Sphinx-based" because the build process can be configured for any kind of documentation stack?

It's still Sphinx-only for now, but we're actively working to remove that limitation.


> Should we clarify in a separate section that the easiest integration happens with Sphinx, but we support any kind of build process and making different formats available is a matter of moving build artifacts into the correct location at the end of a build?

Yes, when it's true :)

Reply:
Ah yeah, referring to the conversation today, the output path of other formats (such as a PDF version) should be accessed through a build environment variable defined in a real build API of sorts?

Is it fine to keep the Sphinx part sort of bit implicit because we're gonna add it very soon?

Reply:
Now I"m confused, since this comment is on a line where I removed the Sphinx-based part?

Reply:
I think your original suggestion should stand :+1: 

To clarify what I mean is that it's fine to remove it *because we're gonna add wider support very soon*. Until then, it can keep being implicit or unspoken whether it's only for Sphinx or not.

* Only Sphinx: Well yes, that's the way it's supported and documented
* Wider support: Well it's actually possible, but it's a longer story that we won't document until it's through an official API.